sdk/textanalytics/ai-text-analytics/review/ai-text-analytics.api.md

```diff
@@ -0,0 +1,792 @@
+## API Report File for "@azure/ai-text-analytics"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { AzureKeyCredential } from '@azure/core-auth';
+import { CommonClientOptions } from '@azure/core-client';
+import { KeyCredential } from '@azure/core-auth';
+import { OperationOptions } from '@azure/core-client';
+import { TokenCredential } from '@azure/core-auth';
+
+// @public
+export interface ActionCommon {
+    disableServiceLogs?: boolean;
+}
+
+// @public
+export type ActionPrebuilt = ActionCommon & {
```

Comment:
extends?

Reply:
I relied heavily on swagger transforms for things that can be easily transformed and these types are some of the results of doing that. I am not sure why code gen uses `&` instead of `extends`, is it just to reduce syntactic overheads?